robot/Cumulus/resources/NPSPSettingsPageObject.py

```diff
@@ -247,4 +247,25 @@ def verify_status_to_state_mappings(self, **kwargs):
                 print(f"actual mapping found for {key} is {actual_value}")
                 assert value == actual_value, "Expected {} value to be {} but found {}".format(key,value, actual_value)
             else:
-                print("Right keys under status mapping fields table not found")    
\ No newline at end of file
+                print("Right keys under status mapping fields table not found")
+
+    @capture_screenshot_on_error
+    def set_batch_number_format_status(self,number_format,status):
+        """Deactivates the given batch number format by clicking on dropdown and selecting option
```

Comment:
This method deactivates and activates both depending on the status . The description talks about only deactivation. Please update to reflect both.

Reply:
thanks for catching, I modified after creating it and forgot to update the documentation